package.json

```diff
@@ -36,7 +36,7 @@
     "test:debug": "mocha debug test/runner.js --reporter spec"
   },
   "dependencies": {
-    "broccoli-caching-writer": "^0.5.4",
+    "broccoli-caching-writer": "^1.1.0",
```

Comment:
https://github.com/ember-cli/ember-cli/blob/master/lib/broccoli/ember-app.js#L1109-L1160 decides what is provided to the css preprocessors